crypto/evp/keymgmt_lib.c

```diff
@@ -429,6 +429,59 @@ int evp_keymgmt_util_copy(EVP_PKEY *to, EVP_PKEY *from, int selection)
     return 1;
 }
 
+/* This is the provided variant of EVP_PKEY_set_alias_type() */
+int evp_keymgmt_util_convert(EVP_PKEY *target, const char *keytype,
+                             OPENSSL_CTX *libctx, const char *propq)
+{
+    EVP_KEYMGMT *new_keymgmt;
+    EVP_PKEY tmp = { 0, };
```

Comment:
Drop the initialisation, `tmp = *target;` below makes it unnecessary.

Reply:
Done